cookbooks/bcpc-hadoop/recipes/disks.rb

```diff
@@ -155,7 +155,13 @@
         end
       end
 
-      mount_device = bcpc_uuid_for_device(dev_name) || dev_name
+      uuid_candidate = bcpc_uuid_for_device(dev_name)
+
+      mount_device = if uuid_candidate
+                       "UUID=#{uuid_candidate}"
```

Comment:
`device_type` is probably the correct way to do it. https://docs.chef.io/resource_mount.html#properties

Just had this breaking in my build now.

```
     Chef::Exceptions::Mount
      -----------------------
      Device UUID=53cd9a71-5a7a-45e0-a9a5-cc19a696ee32 does not exist

      Cookbook Trace:
```